<launch>
  <node name="camera_node" pkg="cinema_bot" type="camera_node.py" output="screen"/>
</launch>
